sdk/typescript/dev/node/src/index.ts

```diff
@@ -12,7 +12,7 @@ import { Commands } from "./commands"
 @object()
 class Node {
   @func()
-  version = "18-alpine"
+  version = "lts-alpine"
```

Comment:
That's a good point but this only changes the default version in the module for developing the SDK. The runtime is currently on v20:

https://github.com/dagger/dagger/blob/6fdf164c7f62a8e57d6306ba07692feef1c04501/sdk/typescript/runtime/main.go#L19-L21

@TomChv, doesn't it make sense to bump the runtime to the latest LTS?

Reply:
Good catch! Remember to **always** use actual versions so that `lts` does not get updated to a new versions without us being ready for it.

If possibly, we should also use sha256 pinning so that we:
- do not do extra round-trips to check if the tag changed
- will be resilient to image tag updates

Reply:
Yes, for best performance it should be the exact same as what's in the runtime. Version down to the patch (instead of "lts" or just the major), and digest.